test/integration/revert_int_test.go

```diff
@@ -24,10 +24,7 @@ func (suite *RevertIntegrationTestSuite) TestRevert() {
 
 	// Revert the commit that added urllib3.
 	commitID := "1f4f4f7d-7883-400e-b2ad-a5803c018ecd"
-	cp := ts.SpawnWithOpts(
-		e2e.OptArgs("revert", commitID),
-		e2e.OptAppendEnv(constants.DisableRuntime+"=true"),
-	)
+	cp := ts.Spawn("revert", commitID)
```

Comment:
Should we set the async runtime config value for this test?

Reply:
No, we need it for the later use of `state shell`. Originally I enabled it, but that caused a failure and caused me to file another ticket and put up #3413.